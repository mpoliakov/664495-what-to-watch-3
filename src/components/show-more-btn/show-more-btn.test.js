import React from 'react';
import renderer from 'react-test-renderer';
import ShowMoreBtn from './show-more-btn.jsx';

it(`<ShowMoreBtn/> is rendered correctly`, () => {
  const onClick = jest.fn();

  const component = renderer.create(<ShowMoreBtn onClick={onClick}/>).toJSON();
  expect(component).toMatchSnapshot();
});
