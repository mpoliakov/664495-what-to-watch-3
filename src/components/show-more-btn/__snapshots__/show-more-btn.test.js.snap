// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShowMoreBtn/> is rendered correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[MockFunction]}
    type="button"
  >
    Show more
  </button>
</div>
`;
